test/runtime/unlink.c

```diff
@@ -45,6 +48,53 @@ static void test_unlinkat(int dirfd)
     }
 }
 
+static void test_tmpfile()
+{
+    int fd;
+    struct statfs statbuf;
+    char buf[8192];
+    char vbuf[8192];
+
+    if (mkdir("/tmp", 0777) != 0 && errno != EEXIST) {
+        perror("mkdir /tmp");
+        exit(EXIT_FAILURE);
+    }
+    statfs("/", &statbuf);
+    unsigned long fsfree = statbuf.f_bfree;
+    fd = open("/tmp", O_RDONLY|O_TMPFILE);
```

Comment:
With gcc 8, this fails to build: `/usr/include/x86_64-linux-gnu/bits/fcntl2.h:50:4: error: call to ‘__open_missing_mode’ declared with attribute error: open with O_CREAT or O_TMPFILE in second argument needs 3 arguments`.

Reply:
Thanks, I hadn't seen this error yet.